src/plugins/validation.ts

```diff
@@ -193,6 +193,25 @@ export class ValidationPlugin {
    }
  };
 

```

Comment:
- [ ] Add the appropriate comments above this validation rule that outline the params, expected return object.  See the comments above `isBetween`